.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @Saransh-cpp
\ No newline at end of file
+* @conda-forge/scikit-hep @Saransh-cpp
```

Comment:
Not sure what's wrong here, maybe I can just remove the team from CODEOWNERS and keep it in `recipe/meta.yaml`?

cc: @matthewfeickert

Reply:
c.f. https://github.com/conda-forge/cuda_histogram-feedstock/pull/4#pullrequestreview-2415850847, but basically, the bot had full control, and so the entire repo is only controlled through the files

* `recipe/meta.yaml`
* `conda-forge.yml`

as the bot will rerender everything else and the README gets generated from `recipe/meta.yaml`.